pkgs/development/tools/hobbes/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv, pkgs }:
```

Comment:
We usually import all the necessary arguments not with `pkgs` but with each package showing up in the list here.